tools/opentsdb_restart.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """Restart opentsdb. Called using -XX:OnOutOfMemoryError=<this script>
 
-Because it's calling the 'service opentsdb' command, should be run as root.
+Because it's calling the 'systemctl *action* opentsdb' command, should be run as root.
```

Comment:
Could you add a check to determine if an OS is running the old school *init* vs *systemd* please? We do have users on older systems.

Reply:
They would be 9 or 10 year old OS versions now so I'm just going to merge this.